test/e2e/gardener/shoot/create_and_delete_hibernated.go

```diff
@@ -24,27 +24,43 @@ import (
 
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	e2e "github.com/gardener/gardener/test/e2e/gardener"
+	"github.com/gardener/gardener/test/framework"
 )
 
 var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
-	f := defaultShootCreationFramework()
-	f.Shoot = e2e.DefaultShoot("e2e-hibernated")
-	f.Shoot.Spec.Hibernation = &gardencorev1beta1.Hibernation{
-		Enabled: pointer.Bool(true),
+	var test = func(f *framework.ShootCreationFramework) {
```

Comment:
```suggestion
	test := func(f *framework.ShootCreationFramework) {
```

some for similar occurrences